contracts/RFQ.sol

```diff
@@ -184,6 +187,8 @@ contract RFQ is IRFQ, Ownable, TokenCollector, EIP712 {
             weth.deposit{ value: amount }();
             weth.transfer(to, amount);
         } else {
+            // this branch cannot be covered because we cannot trigger the sendValue internal revert,
```

Comment:
Maybe have the `to` be a contract and revert at `receive` or `fallback`?

Reply:
Adding the test case cannot improve the branch coverage because we cannot trigger the internal error `AddressInsufficientBalance` here.

Reply:
```suggestion
            // this branch cannot be covered because we cannot trigger the AddressInsufficientBalance error in sendValue,
```

Reply:
ACK